### PR TITLE
Update "software" method in Education

### PR DESCRIPTION
**Pull request by phallobst**
See #4410

### CODE REVIEW

File: _data/education.yml [hunk 1]
```diff
@@ -322,7 +323,7 @@ websites:
   - name: Ryerson University
     url: https://www.ryerson.ca
     tfa:
-      - software
+      - totp
       - hardware
```
Comment (Carlgo11) [inline _data/education.yml]:
```suggestion
      - u2f
```

File: _data/education.yml [hunk 2]
```diff
@@ -379,7 +381,7 @@ websites:
     tfa:
       - sms
       - phone
-      - software
+      - proprietary
       - hardware
```
Comment (Carlgo11) [inline _data/education.yml]:
```suggestion
      - u2f
```

File: _data/education.yml [hunk 3]
```diff
@@ -389,7 +391,7 @@ websites:
...
```
Comment (Carlgo11) [inline _data/education.yml]:
```suggestion
      - u2f
```

File: _data/education.yml [hunk 4]
```diff
@@ -408,7 +410,7 @@ websites:
     img: uw.png
     tfa:
       - phone
-      - software
+      - proprietary
       - hardware
```
Comment (Carlgo11) [inline _data/education.yml]:
```suggestion
      - u2f
```

File: _data/education.yml [hunk 5]
```diff
@@ -419,7 +421,7 @@ websites:
     tfa:
       - sms
       - phone
-      - software
+      - proprietary
       - hardware
```
Comment (Carlgo11) [inline _data/education.yml]:
```suggestion
      - u2f
```

File: _data/education.yml [hunk 6]
```diff
@@ -429,6 +431,6 @@ websites:
...
```
Comment (Carlgo11) [inline _data/education.yml]:
```suggestion
      - u2f
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by Carlgo11** [comment]
I have no idea how the Halifax Bank Twitter handle suddenly starts blocking the build 😕

**Comment by Carlgo11** [comment]
Updating the branch seems to have worked 🤷‍♂

**Review by Carlgo11 — APPROVED** [review]

**Review by Jawshy — APPROVED** [review]
All English websites ✅

**Pull request merged**